compiler/rustc_interface/src/interface.rs

```diff
@@ -347,7 +356,7 @@ pub struct Config {
     /// enabled. Makes it so that "please report a bug" is hidden, as ICEs with
     /// internal features are wontfix, and they are usually the cause of the ICEs.
     /// None signifies that this is not tracked.
-    pub using_internal_features: Arc<std::sync::atomic::AtomicBool>,
```

Comment:
There hasn't been an option here in a while now

Reply:
What do you mean?

Reply:
Hmm misclicked. The doc comment mentions `None`

Reply:
Removed that sentence.